game_patch/multi/server.cpp

```diff
@@ -713,6 +713,16 @@ CodeInjection multi_level_init_injection{
     },
 };
 
+CallHook<rf::Entity* (int, const char*, int, const rf::Vector3&, rf::Matrix3&, int, int)> player_create_entity_entity_create_hook{
+    0x004A41D3,
+    [] (const int type, const char* const name, const int parent_handle, const rf::Vector3& pos, rf::Matrix3& orient, int create_flags, const int mp_character) {
+        if (get_af_remote_info().has_value() && get_af_remote_info().value().no_player_collide) {
+            create_flags |= 0x4;
```

Comment:
It would be nice to put an enum somewhere in `rf/` or at least add a comment with enum member name here